docs/_docs/plugins/remote_build/README.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Plugin: Custom Builder and Storage"
+pdf: true
+toc: true
+permalink: docs/plugins/remote-build
```

Comment:
Does this require a permalink, and if so, should it end in trailing slash?

Reply:
I don't understand :-( !